### PR TITLE
 #168360652 refactoring

### PR DESCRIPTION
**Pull request by irfiacre**
#### What does this PR do?
 This PR adds a new and modified files  for refactoring .
#### Description of Task to be completed?
 - adding modified pages
 - adding defaultValues.js file to help in setting up an admin for testing purposes.
 - deleted files that do not relate to the current action. mentors,
#### How should this be manually tested?
 - First, you clone the repo on your machine.
 - Open the folder in your text editor,  *preferably   `visual studio code`*
 - Now open your terminal, *for `visual studio code`    just click    `ctrl+/`* 
 - After  creating a table with by running  `npm create-tables`,  
 - Then you have to run a command  `npm run dev`.
 - After this, you can view refactored endpoints.

#### What are relevant pivotal tracker stories.

https://www.pivotaltracker.com/story/show/168360652

### CODE REVIEW

File: server/controllers/mentors.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                lastName : Check.lastName,
+                email : Check.email, 
+                address : Check.address,
+                bio : Check.bio,
+                occupation : Check.occupation,
+                expertise : Check.expertise,
+            }
+
+            return res.status(200).json({
+                status: 200,
```
Comment (Kasulejoseph) [inline server/controllers/mentors.js]: This is missing a message

File: server/controllers/mentors.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    const mentors =`SELECT * FROM USERS WHERE is_mentor ='true'`;
+        const {rows} = await pool.query(mentors)
+  
+    const Check = rows.find((objectof) => objectof.id === parseInt(req.params.mentorId));
+
+    if(!Check){
+        return res.status(404).json({
+            status:404,
+            message: "Mentor not found"
+        })
```
Comment (Kasulejoseph) [inline server/controllers/mentors.js]: The key should be error not message

File: server/controllers/mentors.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                   status: 208,
+                   message:"User is already mentor"
+               })
+              }
+           const newInfo= `UPDATE users SET is_mentor=true WHERE id = ${req.params.userId} `
+           const {row} = await pool.query(newInfo)
+      
+           res.status(200).json({
+               status:200,
+               message:"User account changed to mentor",
```
Comment (Kasulejoseph) [inline server/controllers/mentors.js]: Return the `mentor id` and `is_mentor` as the response

File: server/models/defaultValues.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+
+import pool from "../configurations/db-config"
+
+const inserter = `
+  INSERT INTO users(
+    firstName,lastName,email,password,address,bio,occupation,expertise,is_mentor,is_admin
+  )VALUES(
+    'IRANZI',
```
Comment (Kasulejoseph) [inline server/models/defaultValues.js]: Why do you have this dummy data?

Comment (irfiacre) [reply server/models/defaultValues.js]: For running tests!

Comment (Kasulejoseph) [reply server/models/defaultValues.js]: You need to create a test db, So this should be inserted in the test db

File: server/controllers/usersign.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
+import bcrypt from 'bcrypt';
+import '@babel/plugin-transform-regenerator';
+import '@babel/polyfill';
```
Comment (Kasulejoseph) [inline server/controllers/usersign.js]: What do you use babel-polyfill for? I guess it serves no purpose here

Comment (irfiacre) [reply server/controllers/usersign.js]: This helps me  to use the full set of ES6 features beyond syntax changes.

Comment (irfiacre) [reply server/controllers/usersign.js]: But here I imported it expecting to use it. But It's not necessary.

**Pull request merged**